survey-runner-application/terraform.tfvars.example

```diff
@@ -10,7 +10,7 @@ certificate_arn="arn:aws:iam::XXXXXXX:server-certificate/NAME"
 
 # Environment variables
 application_secret_key="you'll never guess it"
-eq_sr_log_level="DEBUG"
+eq_log_level="DEBUG"
```

Comment:
if someone uses this example file will it override the level to DEBUG? therefore not default it to INFO.